src/common/mangataHelper.js

```diff
@@ -178,8 +157,14 @@ class MangataHelper {
 
   calculateRewardsAmount = async (address, symbol) => {
     const liquidityTokenId = this.getTokenIdBySymbol(symbol);
-    const response = await this.mangata.calculateRewardsAmount(address, liquidityTokenId);
-    return response.toNumber();
```

Comment:
response.toNumber() will err because the interger is too big( more than 53 bits).